application/libraries/UBLtemplates/Ublexam20Xml.php

```diff
@@ -0,0 +1,265 @@
+        $node->appendChild($this->xmlCustPostalAddress());        
+        return $node;
+    }
+
+    protected function xmlCustPartyIdentification()
+    {              
+        $node = $this->doc->createElement('cac:PartyIdentification'); 
+        $nodeID = $this->doc->createElement('cbc:ID', $this->invoice->client_vat_id);         
+        $nodeID->setAttribute('schemeAgencyID', $this->invoice->client_country);        
+        $nodeID->setAttribute('schemeAgencyName', 'Example');   
+        $node->appendChild($nodeID);                    
+        return $node;             
+    }
+
+    protected function xmlCustPartyName()
+    {              
+        $node = $this->doc->createElement('cac:PartyName');  
+        $nodeName = $this->doc->createElement('cbc:Name', $this->invoice->client_name);         
+        $node->appendChild($nodeName);                    
+        return $node;             
+    }    
+
+    protected function xmlCustPostalAddress()
+    {              
+        $node = $this->doc->createElement('cac:PostalAddress');  
+        $node->appendChild($this->doc->createElement('cbc:StreetName', $this->invoice->client_address_1));
+        $node->appendChild($this->doc->createElement('cbc:CityName', $this->invoice->client_city));
+        $node->appendChild($this->doc->createElement('cbc:PostalZone', $this->invoice->client_zip));
+        $nodeCountry = $this->doc->createElement('cac:Country');
+        $nodeCountry->appendChild($this->doc->createElement('cbc:IdentificationCode', $this->invoice->client_country));
+        $node->appendChild($nodeCountry);                   
+        return $node;             
+    }   
+
+    //PaymentMeans
+    protected function xmlPaymentMeans()
+    {   
+        $PaymentDueDate = $this->invoice->invoice_date_due; 
+        $node = $this->doc->createElement('cac:PaymentMeans');    
+        $nodePMC = $this->doc->createElement('cbc:PaymentMeansCode', '1'); 
+        $nodePMC->setAttribute('listID', 'UN/ECE 4461'); 
+        $nodePMC->setAttribute('listName', 'Payment Means'); 
+        $node->appendChild($nodePMC);                                       
+        if ($PaymentDueDate) {$node->appendChild($this->doc->createElement('cbc:PaymentDueDate', $PaymentDueDate));} 
+        $node->appendChild($this->xmlPFAccount());        
+        return $node;            
+    }
+
+    protected function xmlPFAccount()
+    {   
+        $BankRekIBAN = $this->invoice->user_iban;       
+        $node = $this->doc->createElement('cac:PayeeFinancialAccount');     
+        $nodeID = $this->doc->createElement('cbc:ID', $BankRekIBAN);
+        $nodeID->setAttribute('schemeName', 'IBAN');  
+        $node->appendChild($nodeID);        
+        return $node;
+    }
+
+    //PaymentTerms
+    protected function xmlPaymentTerms()
+    {    
+        $date = date_create($this->invoice->invoice_date_due);
+        $PaymentTerms = 'Pay before: ' . date_format($date, 'd/m/Y');
```

Comment:
Is is required to use `d/m/Y` as the date format instead of using a ISO format or the user-defined one?